feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.spec.ts

```diff
@@ -781,7 +791,78 @@ describe('ConfigAttributeHeaderComponent', () => {
       expect(component['logWarning']).toHaveBeenCalled();
     });
 
-    it('should call focusAttribute', () => {
+    it('should navigate from conflict group to regular group that contains attribute with selected value which is involved in conflict', () => {
+      component.groupType = Configurator.GroupType.CONFLICT_GROUP;
+      component.attribute.groupId = ConfigurationTestData.GROUP_ID_2;
+
+      const value1 = createValue('1', 'val1', false);
+      const value2 = createValue('2', 'val2', false);
+      const value3 = createValue('3', 'val3', true);
+
+      component.attribute.values = [value1, value2, value3];
+
+      spyOn(configurationGroupsService, 'navigateToGroup');
+      fixture.detectChanges();
+
+      component.navigateToGroup();
+      expect(configurationGroupsService.navigateToGroup).toHaveBeenCalledTimes(
+        1
+      );
+    });
+
+    it('should navigate from conflict group to regular group that contains attribute without any selected value which is involved in conflict', () => {
+      component.groupType = Configurator.GroupType.CONFLICT_GROUP;
+      component.attribute.groupId = ConfigurationTestData.GROUP_ID_2;
+
+      const value1 = createValue('1', 'val1', false);
+      const value2 = createValue('2', 'val2', false);
+      const value3 = createValue('3', 'val3', false);
+
+      component.attribute.values = [value1, value2, value3];
+
+      spyOn(configurationGroupsService, 'navigateToGroup');
+      fixture.detectChanges();
+
+      component.navigateToGroup();
+      expect(configurationGroupsService.navigateToGroup).toHaveBeenCalledTimes(
+        1
+      );
+    });
```

Comment:
same as above. test does not test written code. expect focusValue was called is missing